tenable/io/v3/was/scans/api.py

```diff
@@ -0,0 +1,230 @@
+'''
+Scans
+=====
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`Web Application Scanning v3 scans <was-v2-scans>` API.
+
+Methods available on ``tio.v3.was.scans``:
+
+.. rst-class:: hide-signature
+.. autoclass:: ScansAPI
+    :members:
+'''
+from io import BytesIO
+from typing import Dict, Optional
+from uuid import UUID
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.was.scans.schema import ScanReportSchema, ScanStatusSchema
+
+
+class ScansAPI(ExploreBaseEndpoint):
```

Comment:
Are we able to make a base class that both VM and WAS can use?

Reply:
Most of the methods under VM and WAS. So we are keeping both of them different. Eg. get notes and search vulnerabilities in was/scans and many more endpoints under vm/scans are different. 
Although common methods such as details and search are covered in ExploreBaseEndpoint class which is inherited by every module.